posthog/hogql_queries/web_analytics/web_overview_pre_aggregated.py

```diff
@@ -1,61 +1,26 @@
 from typing import TYPE_CHECKING, cast
-from datetime import datetime, UTC
 
 from posthog.hogql import ast
 from posthog.hogql.parser import parse_select
+from posthog.hogql_queries.web_analytics.pre_aggregated.query_builder import WebAnalyticsPreAggregatedQueryBuilder
 
 if TYPE_CHECKING:
     from posthog.hogql_queries.web_analytics.web_overview import WebOverviewQueryRunner
 
+SUPPORTED_PROPERTIES = {
+    "$host": "host",
+    "$device_type": "device_type",
+}
```

Comment:
Can we make this a shared constant rather than duplicating it in here + the other file? This implies consistency is easier to guarantee, while the comments-approach we have is very error-prone

Reply:
I like the idea, but I don't intend to keep them equal for more than one or two days, though. Just enough to test the prod behavior while checking the other moving parts